### ISSUE TITLE
Update Twisted Web client howto to cover cookies

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5110]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5110|
|Type|enhancement|
|Created|2011-05-15 13:47:35Z|
|Branch|https://github.com/twisted/twisted/tree/cookieagent-howto-5110|

[#4922](https://github.com/twisted/twisted/issues/4922) added support for accepting and sending cookies (and keeping track of them meanwhile).  The client howto should talk about how to use this feature.

<details><summary>Searchable metadata</summary>

```
trac-id__5110 5110
type__enhancement enhancement
reporter__exarkun exarkun
priority__high high
milestone__ 
branch__branches_cookieagent_howto_5110 branches/cookieagent-howto-5110
branch_author__jonathanj jonathanj
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1305467255000000 1305467255000000
changetime__1309193762000000 1309193762000000
version__None None
owner__jonathanj jonathanj
cc__jknight cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_2"></a>|@jonathanj commented|
|-|-|

(In [32188]) Branching to 'cookieagent-howto-5110'

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_3"></a>|@jonathanj commented|
|-|-|

(In [32189]) Add CookieAgent section to twisted.web.client howto, refs [#5110](https://github.com/twisted/twisted/issues/5110).

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_4"></a>|@jonathanj commented|
|-|-|

Buildbot results: http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/cookieagent-howto-5110

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @jonathanj|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_6"></a>|@jonathanj set status to `closed`|
|-|-|

(In [32195]) Merge cookieagent-howto-5110.

Author: jonathanj
Reviewer: thijs
Fixes: [#5110](https://github.com/twisted/twisted/issues/5110)

Add CookieAgent documentation to the twisted.web.client how-to.
